requirements-dev.txt

```diff
@@ -2,6 +2,7 @@
 
 # For running doctests (during "python setup.py test")
 Sphinx >= 1.4.5
+sphinx_bootstrap_theme
```

Comment:
I believe that we agreed several weeks ago that the purpose of requirement files are:

- requirements.txt: The app user must install it
- requirements-dev.txt: The app developer must install it (to develop, test and lint)
- requirements-doc.txt: Whoever is going to change/generate the docs should install it

For example, only code developer should install linters and only doc editors should install sphinx-autobuild.